packages/navigator/src/browser/navigator-model.ts

```diff
@@ -94,7 +94,7 @@ export class FileNavigatorModel extends FileTreeModel {
 
     previewNode(node: TreeNode): void {
         if (FileNode.is(node)) {
-            open(this.openerService, node.uri, { mode: 'reveal', preview: true });
+            open(this.openerService, node.uri, { preview: true });
```

Comment:
I took a deeper look at the behaviour of the file/editor selection itself.
Depending on the preference settings, the editor can either be revealed or opened (and therefore automatically activated).
Regarding the `revealed` mode (which is currently the case for the preferences `Editor > Enable Preview > true` as well as `Workbench > List > Open Mode > singleClick`):
If the editor is revealed, the global selection is not updated and does not contain the revealed editor, as the file node in the navigator stays the current selection. (This is also observable in the GIF from your first [comment](https://github.com/eclipse-theia/theia/pull/8655#pullrequestreview-515724579))

Therefore I would say the behaviour is technically correct as the `property-view` listens to the current selection,
but I agree that it might not be very intuitive to the user.

I can think of two possible ways to go on from here:
- Stick to the behavioural change and activate editors if they are opened via `singleClick` or in `preview` to assure that the global selection is up to date.
- Adapt the `EditorWidget` and `EditorPreviewWidget` to set the global selection if they are revealed.

What's your opinion on this? Thanks in advance.

Reply:
> I took a deeper look at the behaviour of the file/editor selection itself.
> Depending on the preference settings, the editor can either be revealed or opened (and therefore automatically activated).
> Regarding the `revealed` mode (which is currently the case for the preferences `Editor > Enable Preview > true` as well as `Workbench > List > Open Mode > singleClick`):
> If the editor is revealed, the global selection is not updated and does not contain the revealed editor, as the file node in the navigator stays the current selection. (This is also observable in the GIF from your first [comment](https://github.com/eclipse-theia/theia/pull/8655#pullrequestreview-515724579))
> 
> Therefore I would say the behaviour is technically correct as the `property-view` listens to the current selection,
> but I agree that it might not be very intuitive to the user.
> 
> I can think of two possible ways to go on from here:
> 
> * Stick to the behavioural change and activate editors if they are opened via `singleClick` or in `preview` to assure that the global selection is up to date.

I do not believe we should modify the current `activate` behavior.

> * Adapt the `EditorWidget` and `EditorPreviewWidget` to set the global selection if they are revealed.

This may be the correct behavior, although I'm not sure myself as I have yet to verify if there are any implications when setting the global selection in this case. Would it be possible to rely on the `shell` to determine the current widget and update the `property-view` then? 

- https://eclipse-theia.github.io/theia/docs/next/classes/core.applicationshell-2.html#onDidChangeCurrentWidget

Reply:
I agree with you and reverted the current `activate` behaviour.
The combination of selection and focus is the main point to look at in my opinion. As it depends on the settings, how editors are opened, I think sticking to the global selection and focus without relying on changes regarding the `ApplicationShell` might be the cleanest way for the `property-view` overall.
Please see my attached screencast, where I show the differences between activated editors and 'only revealed' editor. If editors are revealed, the focus stays in the file navigator widget, until the user decides to activate an editor (or any other widget). This would mean that the `property-view` would depend only on the global selection/focus behaviour and would align without adaptions if there will be changes in the future on that matter.

![global-selection-focus](https://user-images.githubusercontent.com/47414588/102607804-3561d280-4129-11eb-9c75-02f5056bafae.gif)